src/nhp/docker/run.py

```diff
@@ -7,17 +7,18 @@
 import logging
 import os
 import re
-import threading
 from pathlib import Path
 from typing import Any, Callable
 
 from azure.identity import DefaultAzureCredential
 from azure.storage.blob import BlobServiceClient
 from azure.storage.filedatalake import DataLakeServiceClient
 
-import config
-from model.helpers import load_params
-from run_model import run_all
+from nhp.docker import config
+from nhp.model.__main__ import (
+    run_all,  # FIXME: Not a good idea to import from __main__.py
```

Comment:
```suggestion
    run_all,  # TODO: Not a good idea to import from __main__.py
```

for consitency,  elsewhere we have used TODO's rather than FIXME's. we have a step in our release process to search for TODO's so this would be missed